cypress/support/commands.js

```diff
@@ -30,11 +30,26 @@ Cypress.Commands.add("reLogin", (userMail, userPassword) => {
   cy.get("div[data-cy=email] input").type(`${userMail}`);
   cy.get("div[data-cy=password] input").type(`${userPassword}`);
   cy.get("button[data-cy=signInButton]").click();
+  cy.location("pathname", { timeout: 10000 }).should("not.include", "/signin");
 });
 
 Cypress.Commands.add("openAppDrawer", () => {
-  cy.get("button[data-cy=appDrawerOpener]").click();
-  cy.get("div[data-cy=appDrawerDiv]").should("be.visible");
+  cy.get("button[data-cy=appDrawerOpener]").should("be.visible");
+  cy.get("body").then($body => {
```

Comment:
This is in a commit called "fix: cypress linter" but it's not very clear why. Perhaps in your commit explain why we're doing this and why the drawer isn't always visible?

Reply:
Yeah, I don't even understand what happened. I've done` git add -all`, then written commit message and pushed...then linter started to complain(maybe already after commit)..some code got pushed and some didn't even get commited and was only staged. Really no idea what happened..but the changes in those commits don't necessarily fit commit messages.

Reply:
It's totally weird...it seems the first commit is "fix: cypress linter" commit...soo nothing got commited during my first try...dunno why I thought it did :confused:

Reply:
Done.